### PR TITLE
Fixes broken question list plugin, lite refactoring

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: aldryn_faq/models.py [hunk 1]
```diff
@@ -189,15 +190,19 @@ class Meta:
...
     def get_questions(self):
         return self.questions.all()
 
+    def __str__(self):
+        question_count = self.questions.count()
+        return '{0} {1} selected'.format(
+            question_count,
+            "question" if question_count == 1 else "questions"
+        )
+
```
Comment (mkoistinen) [inline aldryn_faq/models.py]: This is the fix, here.

Comment (czpython) [inline aldryn_faq/models.py]: wouldn't it be better to use ```from django.template.defaultfilters import pluralize```?

Comment (mkoistinen) [inline aldryn_faq/models.py]: Seems overkill to me and a misuse of a template filter. =)

Comment (mkoistinen) [inline aldryn_faq/models.py]: But, I should probably make this translatable.

Comment (czpython) [inline aldryn_faq/models.py]: maybe.. but i don't I think is better to reuse but not required.

I wouldn't make it translatable as this is used for the object's internal representation, not very user facing at least not all the time.

**Comment by czpython** [comment]
:+1:

**Pull request merged**